### ISSUE TITLE
Version 1.4.2 shows as 1.4.1

### ISSUE THREAD

**Issue by martin-rueegg** [nerzhul/ocsms#41]
appinfo/info.xml should read
```
<?xml version="1.0"?>
<info>
    <id>ocsms</id>
    <name>ownCloud SMS</name>
    <description>A app to sync SMS with your ownCloud</description>
    <licence>AGPL</licence>
    <author>Loic Blot</author>
    <version>1.4.2</version>
    <requiremin>7</requiremin>
</info>
```

**Comment by nerzhul** [comment]
hello,
sorry i don't understand what you mean here ?

**Comment by martin-rueegg** [comment]
sorry, i made a mistake in the issue title! :-)

the version string is set to **1.4.1** instead of **1.4.2**

**Comment by nerzhul** [comment]
Right... i fix it now

**Comment by nerzhul** [comment]
Fixed, we bump to 1.4.3
